client/client.go

```diff
@@ -547,46 +651,41 @@ func (c *client) handleDispatcher(dispatcherCtx context.Context, dc string, tsoD
 				continue
 			}
 		}
+		if err = tbc.fetchPendingRequests(dispatcherCtx); err != nil {
+			log.Error("[pd] fetch pending tso requests error", zap.String("dc-location", dc), errs.ZapError(errs.ErrClientGetTSO, err))
+			return
+		}
+		tbc.adjustBestBatchSize()
```

Comment:
should consider the `maxBatchWaitInterval` = 0. when disable batch strategy, it should same as before.